quantities/units/Imperial/Temperature/Farenheit.cs

```diff
@@ -6,9 +6,6 @@ namespace Quantities.Units.Imperial.Temperature;
 // See: https://en.wikipedia.org/wiki/Conversion_of_units#Temperature
 public readonly struct Fahrenheit : IImperialUnit, ITemperature
 {
-    private const Decimal kelvinOffset = 459.67m;
-    private const Decimal scaleFromKelvin = 9m / 5m;
-    public static Double ToSi(in Double nonSiValue) => (Double)(5m * ((Decimal)nonSiValue + kelvinOffset) / 9m);
```

Comment:
We can't use `Decimals`for our conversions anymore. This partially explains the loss of precision when additions and subtractions are needed for conversions.

But now, this can all be fixed within the library itself and does not rely on the specifics of these `ToSi` and `FromSi` implementations on the units.